notebooks/notebooks/paper/estimate-gpu-days-adv.ipynb

```diff
@@ -32,6 +32,7 @@
     "import matplotlib.pyplot as plt\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #43.        plt.show()
</pre><p>The large amount of left padding on "Victim visits" in the legend looks a little odd to me, but not sure how easy that is to adjust</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlignmentResearch/KataGoVisualizer/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>If I try to make it look right in the notebook, it will actually look messed up in the paper. I think the paper renderer and the notebook renderer just differ slightly.</p>